tools/github-promote-release.ts

```diff
@@ -14,39 +14,42 @@ type Options = {
   platform: 'web' | 'android' | 'ios'
 }
 
-const getReleaseIdBody = async ({ deliverinoPrivateKey, owner, repo, platform }: Options) => {
+const getReleases = async ({ deliverinoPrivateKey, owner, repo, platform }: Options) => {
   const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
 
   const releases: Releases = await appOctokit.rest.repos.listReleases({
     owner,
     repo,
   })
+  return releases.data.filter(release => release.tag_name.includes(platform))
+}
 
-  const result = releases.data.find(release => release.tag_name.includes(platform))
-  if (result && result.prerelease) {
-    console.warn('Unset prerelease tag of ', result.tag_name)
-    return result
-  }
+const promoteReleases = async ({ deliverinoPrivateKey, owner, repo, platform }: Options) => {
+  const releases = await getReleases({ deliverinoPrivateKey, owner, repo, platform })
+  const preReleases = releases.filter(release => release.prerelease)
 
-  console.warn('No release found to unset the prerelease tag for. Latest release may already be non-prerelease')
-  return null
-}
+  const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
+  await Promise.all(
+    preReleases.map(async preRelease => {
+      const result = await appOctokit.rest.repos.updateRelease({
+        owner,
+        repo,
+        release_id: preRelease.id,
+        prerelease: false,
+        make_latest: platform === 'android' ? 'true' : 'false', // We always want android to be the latest release, so a link to the latest github release will go to the apk
+      })
+      console.warn('Http response code of updating the result: ', result.status)
+    }),
+  )
 
-const removePreRelease = async ({ deliverinoPrivateKey, owner, repo, platform }: Options) => {
-  const release = await getReleaseIdBody({ deliverinoPrivateKey, owner, repo, platform })
-  if (!release) {
-    return null
+  const filteredTagNames = preReleases.map(release => release.tag_name)
+  console.warn('Unset prerelease tags of ', filteredTagNames)
```

Comment:
:upside_down_face: Optional: How about moving this to L41 inside the `.map()`?
e.g. something like this:
``` ts
      console.warn(`Release ${release.tag_name} promoted with status:`, result.status)
```